### ISSUE TITLE
Get Index of an Entity in a Sentence

### ISSUE THREAD

**Issue by iCHAIT** [explosion/spaCy#4175]
Hi,

I want to know if there is an elegant way to get the index of an Entity with respect to a Sentence. I know I can get the index of an Entity in a string using `ent.start_char` and `ent.end_char`, but that value is with respect to the entire string.

For example in the following code - 

```python

import spacy

nlp = spacy.load("en_core_web_sm")
doc = nlp(u"Apple is looking at buying U.K. startup for $1 billion. Apple just launched a new Credit Card.")

for ent in doc.ents:
    print(ent.text, ent.start_char, ent.end_char, ent.label_)
```

I want the Entity `Apple` in both the sentences to point to start and end indexes 0 and 5 respectively.

Basically, I am trying to achieve similar to this [SO Question](https://stackoverflow.com/questions/50742516/how-to-get-the-index-of-a-token-in-a-sentence-in-spacy) but for Entities rather than tokens.

**Comment by ines** [comment]
This question is already on Stack Overflow as well, so I'm closing this issue in favour of the SO thread: https://stackoverflow.com/questions/57606940/how-to-get-index-of-an-entity-in-a-sentence-in-spacy

Since this is a general usage question, Stack Overflow is typically a much better place to ask 🙂 We try to keep this issue tracker focused on bug reports and feature requests.

**Comment by iCHAIT** [comment]
Thanks @ines.

I'll keep in mind for the future.
